crates/typst-kit/src/package.rs

```diff
@@ -109,6 +124,8 @@ impl PackageStorage {
             // version.
             self.download_index()?
                 .iter()
+                .lazy_deser::<PackageInfo>()
```

Comment:
since we only use name & version for this function, it might be nice to use a smaller type with just these two fields here to be resilient to widening type changes to `PackageInfo`'s other field